web/react/components/error_bar.jsx

```diff
@@ -31,6 +61,14 @@ export default class ErrorBar extends React.Component {
         ErrorStore.removeChangeListener(this.onErrorChange);
     }
 
+    componentWillUpdate(nextProps, nextState) {
+        if (this.isValidError(nextState)) {
```

Comment:
Haha. Well when I commented the first time it wasn't accessing the DOM. 
Shouldn't this be in `componentDidUpdate`?